boms/user/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source
+  ~ Copyright 2018, Red Hat, Inc., and individual contributors as indicated
+  ~ by the @authors tag.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>30.0.0.Beta1-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <groupId>org.wildfly.bom</groupId>
+    <artifactId>wildfly</artifactId>
+
+    <packaging>pom</packaging>
+
+    <name>WildFly BOMs: Parent Aggregator</name>
```

Comment:
We should really try hard to change this name and description.
More so that in contrast to other modules at `/boms`, this new one doesn't contain BOMs per se, it contains bom-builder projects so its name could be misleading.

Reply:
I would prefer to not change this for now, it's the WildFly BOMs project that may be found in this directory (as stated by the README too), being this the parent aggregator module. If you look down the tree then builders are identified as such in name and artifactId too.

Reply:
fair enough, my point stands, but leaving it up to other reviewers then.

Reply:
I think it's a fair point but I'd like to leave it for later.

With this our names will be one of "WildFly: xxx" "WildFly Preview: xxx" or "WildFly BOMs: xxx". Those all relate to different end-user consumables, so it's a reasonable categorization.

I wouldn't want to change the name of the other bom modules to "WildFly BOMs: xxx" as that means all sorts of other groupings are ok too, e.g. "WildFly Clustering: xxx". And that will just become a mess (as it was until sometime in the last year or so when we did some housekeeping.)